platformerLib/GreySquareActor.cpp

```diff
@@ -63,8 +63,10 @@ void GreySquareActor::ApplyMoveSpeed(const oxygine::UpdateState& us)
 void GreySquareActor::Jump()
 {
    auto speed = GetSpeed();
-   if(speed.y > 0 * kPixelPerSecond) speed.y *= 0;
-   SetSpeed(speed + kJumpSpeed);
+   if (speed.y.Value() == 0) {
```

Comment:
I've fixed the first item.
According to the second one - let's discuss it separately.